core/src/parser.rs

```diff
@@ -460,6 +464,12 @@ fn get_field_type_override(attrs: &[syn::Attribute]) -> Option<String> {
     attr_value("typeshare", attrs, PREFIX, SUFFIX)
 }
 
+fn get_field_readonly(attrs: &[syn::Attribute]) -> bool {
+    const PREFIX: &str = "typescript";
+    const SUFFIX: &str = "(readonly)";
+    attr_value("typeshare", attrs, PREFIX, SUFFIX).is_some()
+}
```

Comment:
Ideally we should have a more extendable way to handle language-specific parameters like this. If we ever add future configuration here, this code will break.

This ties back into #63, which is also about implementing language-specific features. I was going to do some refactoring of our attribute code to support this, so maybe we could combine our work here?

Reply:
Yes, I noticed it's rather hack-ish when I implemented it and hoped for some ideas during review. Thank you @InquisitivePenguin for sharing. I'll gladly rebase this PR on top of a more generic solution.

Reply:
Thanks! Work to update the parser is happening in https://github.com/1Password/typeshare/pull/66

Reply:
The parser work has been merged into main, so you should be able to update this PR accordingly once you pull from main.

Reply:
Thanks for the notification @InquisitivePenguin!